<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Thien-Tran" ignored="0" name="src/test/java" tests="10" failures="4" timestamp="26 Nov 2018 15:54:37 GMT" time="49.044" errors="0">
  <testcase name="TC07" time="32.485" classname="test.register.RegisterTest"/>
  <testcase name="TC10" time="0.338" classname="test.register.RegisterTest">
    <failure type="java.lang.AssertionError" message="Register Account failed. Message should be::: There&amp;apos;re errors in the form. Please correct the errors and try again expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Register Account failed. Message should be::: There're errors in the form. Please correct the errors and try again expected [true] but found [false]
at test.register.RegisterTest.TC10(RegisterTest.java:59)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- TC10 -->
  <testcase name="TC01" time="0.374" classname="Tests.login.LoginTest"/>
  <testcase name="TC02" time="0.184" classname="Tests.login.LoginTest"/>
  <testcase name="TC03" time="0.284" classname="Tests.login.LoginTest">
    <failure type="java.lang.AssertionError" message="Message should be::: There was a problem with your login and/or errors exist in your form. expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Message should be::: There was a problem with your login and/or errors exist in your form. expected [true] but found [false]
at Tests.login.LoginTest.TC03(LoginTest.java:69)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- TC03 -->
  <testcase name="TC04" time="0.133" classname="Tests.login.LoginTest"/>
  <testcase name="TC05" time="0.487" classname="Tests.login.LoginTest">
    <failure type="java.lang.AssertionError" message="Message should be::: You have used 4 out of 5 login attempts. After all 5 have been used, you will be unable to login for 15 minutes. expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Message should be::: You have used 4 out of 5 login attempts. After all 5 have been used, you will be unable to login for 15 minutes. expected [true] but found [false]
at Tests.login.LoginTest.TC05(LoginTest.java:85)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- TC05 -->
  <testcase name="TC06" time="0.651" classname="Tests.login.LoginTest"/>
  <testcase name="TC08" time="0.222" classname="Tests.login.LoginTest"/>
  <testcase name="TC09" time="0.672" classname="test.changePassword.ChangePasswordTest">
    <failure type="java.lang.AssertionError" message="Message should be::: Your password has been updated expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Message should be::: Your password has been updated expected [true] but found [false]
at test.changePassword.ChangePasswordTest.TC09(ChangePasswordTest.java:53)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- TC09 -->
</testsuite> <!-- src/test/java -->
